interview.md

```diff
@@ -11,41 +11,104 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+People often brainstorm ideas by writing down whatever comes to mind and organizing them later. This can be students trying to find ideas and sources for an essay, travelers planning a trip to some destination, or an all-purpose note sheet for random thoughts and to-dos. However, the result of brainstorming is often hard to parse because of information overload. Nonetheless, there is generally some existing organization within the brainstormed document and a desired organization of these ideas post-brainstorming. An example would be turning potential places to visit into a formal itinerary for the traveler's case.
+
```

Comment:
I thought about a similar domain. There are definitely areas to be improved within note taking. It also seems like a topic that would create a great cs111 project.